src/pathwayvis/models/MapDataObject.ts

```diff
@@ -39,6 +39,14 @@ export class MapDataObject implements MapObject {
     this.mapData.objectiveReaction = reaction;
   }
 
+  public setBounds(bounds: number[]): void {
+    this.mapData.bounds = bounds;
```

Comment:
Why is this a list of numbers? Is this all changed bounds of reactions

Reply:
Old code.